examples/scripts/04-ppo-sentiment.py

```diff
@@ -55,8 +55,8 @@
 
 sentiment_pipe = pipeline("sentiment-analysis","lvwerra/distilbert-imdb", device=pipe_device)
 
-gpt2_model = GPT2HeadWithValueModel.from_pretrained(config['model_name'])
-gpt2_model_ref = GPT2HeadWithValueModel.from_pretrained(config['model_name'])
+gpt2_model = AutoModelForCausalLMWithValueHead.from_pretrained(config['model_name'], summary_dropout_prob=0.1)
```

Comment:
shouldn't the dropout still be the same?

Reply:
By default the dropout is set to `None` (Identity) --> https://github.com/younesbelkada/trl/blob/8390e3bfabdb70da558361483981199304095f05/trl/models/modeling_vhead.py#L26
I think we have to set it to `0.1` since by default it was set to `0.1` on the previous script

Reply:
As discussed, let's set the dropout to `0.1` by default , [cf77820](https://github.com/lvwerra/trl/pull/53/commits/cf778206b734d9cefb79e318a4fb936a92c053e5)